src/components/content/heroes/blog-hero.tsx

```diff
@@ -16,7 +17,7 @@
 
 type ImageHeroProps = Partial<HeroWithText> & {
   image: IGatsbyImageData;
-  attribution?: any;
+  attribution?: Omit<ContentfulBlogPostHero, 'image'>;
```

Comment:
this is 100% correct but I don't like the semantic it introduces when reading the code. It emphasizes on `image` while it's exactly the opposite. For the reader it's not clear and I think a dervice type `BlogAttribution` would be much better to read the code. Something like 

`type BlogAttribution = Omit<ContentfulBlogPostHero, 'image'>;`

or even better explicitly pick the interesting fields because I thin kwhenever something is added it's mostly NOT attribution related so the assumption of omiting is wrong here, isn't it?